app/src/test/java/org/mozilla/fenix/components/metrics/GleanMetricsServiceTest.kt

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.metrics
+
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import mozilla.components.service.glean.testing.GleanTestRule
+import mozilla.components.support.test.robolectric.testContext
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.fenix.GleanMetrics.Events
+import org.mozilla.fenix.GleanMetrics.Metrics
+import org.mozilla.fenix.GleanMetrics.SearchDefaultEngine
+import org.mozilla.fenix.TestApplication
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@ObsoleteCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
+class GleanMetricsServiceTest {
+    @get:Rule
+    val gleanRule = GleanTestRule(testContext)
+
+    @Test
+    fun `setStartupMetrics sets some base metrics`() {
+        val gleanService = GleanMetricsService(testContext)
```

Comment:
Perhaps we should swap this for a `private lateinit var gleanService: GleanMetricsService`
and add an `@Before` that inits it? I'm assuming we'll want to use it in most tests here and we're already using it in 100% of the tests in this file :)

This would also better pattern match the tests we've been writing recently.

Reply:
Sounds good to me! As long as we don't track any state between tests, that's fine by me!